components/ChangeImage.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components/native";
+import { Text, TouchableOpacity } from "react-native";
+
+const ButtonContainer = styled.TouchableOpacity`
+  width: 80px;
+  height: 80px;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50px;
+  background-color: rgba(0, 0, 0, 0.4);
+`;
+
+const ButtonText = styled.Text`
+  color: #fff;
+  font-size: 16px;
+`;
+
+const ChangeImage = ({
+  text,
+  setImgNumber,
+  setQuery,
+  query,
+  newQuery,
+  random,
+}) => {
+  const handleOnPress = () => {
```

Comment:
Since you use this function in the same component you declare it in, it shouldn't have `handle` in the name. `onButtonPress` or `onButtonClick` would be more appropriate